client/src/components/SignUp.tsx

```
@@ -168,18 +160,21 @@ const SignUp = () => {
-                message: "Password must be at least 6 characters long",
+                message: 'Password must be at least 6 characters long',
               },
             })}
             placeholder="Enter Your Password"
             className="form-input p-2 w-full"
           />
-          <span className="absolute right-3 top-3 cursor-pointer text-gray-600" onClick={togglePasswordVisibility}>
+          <span
+            className="absolute right-3 top-3 cursor-pointer text-gray-600"
+            onClick={togglePasswordVisibility}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)